app/utilities/schema.py

```diff
@@ -46,17 +47,13 @@ def load_schema_file(schema_file, language_code=None):
 
 
 def _get_schema_file_path(schema_file, language_code):
-    if language_code != 'en':
-        schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
-    else:
-        schema_dir = settings.EQ_SCHEMA_DIRECTORY
-
+    schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
     return os.path.join(schema_dir, schema_file)
 
 
 def available_json_schemas():
     files = []
-    for file in os.listdir(settings.EQ_SCHEMA_DIRECTORY):
-        if os.path.isfile(os.path.join(settings.EQ_SCHEMA_DIRECTORY, file)) and file.endswith(".json"):
+    for file in os.listdir(settings.EQ_SCHEMA_DIRECTORY_DEFAULT):
```

Comment:
I think there should be a `_get_schema_path(language)` function that returns this value rather than have a new environment var to do it; basically factor line 50 above out into it's own function to be used here and in `_get_schema_file_path`